src/config/appSettingTypes.ts

```diff
@@ -1,5 +1,13 @@
+import { AppSetting } from '@graasp/apps-query-client';
+
 enum APP_SETTINGS_TYPES {
   FILTERED_MEMBERS = 'filtered-members',
 }
 
+export type FilteredMembersSettingType = AppSetting & {
+  data: {
+    filteredMembers: Array<string>;
```

Comment:
It is interesting to see this syntax, i am more used to `string[]` but according to the [TS docs](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#arrays) they are the same 😊 so no need to change it.

Reply:
This syntax is more fluent if you read the code out loud 😉